### PR TITLE
Adds a new Gremorian infested away site

### PR DESCRIPTION
**Pull request by 4000daniel1**
An asteroid hideout for a shady group of people who unfortunately found out they aren't alone on the asteroid.

My goal was to make a rather challenging away site, that rewards a properly organized expedition team with an intense experience. Very open to any suggestions or changes!

At the time of posting doesn't actually seem to be visitable from my testing. It spawns, but not on the overmap. Not sure why.

### CODE REVIEW

**Comment by 4000daniel1** [comment]
!wip

**Comment by DreamySkrell** [comment]
Cloned it locally to see the mapping. Some thoughts:
1. The loot is way too good. There is lots of weapons, there is at least 20 random loot crates, 6 bluespace crystals, 6 random high value items that can be powerful stuff like a actual breacher rig.
2. Simple NPC mobs are not really all that challenging to fight, and are mostly really just a matter of bringing enough guns and ammunition, and maybe metal for barricades to game the simple AI.
3. We do not really have the syndicate anymore. There are beds with syndicate bedsheets, and also a syndicate beacon that, if I am reading its code correctly, can make any random player a traitor.
4. Lots of phoron sheets scattered around, making it look like a mining station that mines phoron. Phoron is not really supposed to be mineable from any random asteroid in our setting, and only should exist in very specific places.
5. Lots of full phoron canisters there. With the phoron scarcity, these would be the best loot from this site. Consider making them all (or most of them at least) into the phoron_scarce variant.

**Comment by WickedCybs** [comment]
This should show up on the overmap once the landmark is mapped in. I would reccomend changing the name of the map (as it appears on the overmap) to something other than "Lonely Asteroid" though as we have like three or two other maps using the same name atm. 

The map does work though and it's a good showing for your first PR. I'd suggest making the entry-point a proper airlock (that can cycle) or put some inflatables there to have it look run-down. As it currently is, people risk venting a lot of it, which might be fine still.

That said, you should remove the revolver at the least. That's kind of way too much.

File: maps/away/generic/shady.dm [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	name = "shady asteroid"
+	desc = "An asteroid with a hangar carved out inside it. Scans detect an unregistered structure within, with multiple lifeforms present."
+	icon_state = "object"
+
+
+/area/hideout
+	name="hideout outpost"
+	icon_state = "outpost_mine_main"
+	requires_power = FALSE
+
```
Comment (SleepyGemmy) [inline maps/away/generic/shady.dm:24]:
```suggestion
```

File: maps/away/generic/shady.dm [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	descriptor = "An asteroid with a hangar carved out inside it. Scans detect an unregistered structure within, with multiple lifeforms present."
+
+/obj/effect/overmap/visitable/sector/shady
+	name = "shady asteroid"
+	desc = "An asteroid with a hangar carved out inside it. Scans detect an unregistered structure within, with multiple lifeforms present."
+	icon_state = "object"
+
+
+/area/hideout
+	name="hideout outpost"
```
Comment (SleepyGemmy) [inline maps/away/generic/shady.dm:21]:
```suggestion
	name = "hideout outpost"
```

**Review by SleepyGemmy — COMMENTED** [review]
some minor code typos aside, the PR looks fine.
note that the colossal greimorian can take 1100 damage, if this is what you intended.

## mapping feedback
image
missing corner wall.

**Review by SleepyGemmy — APPROVED** [review]

**Comment by 4000daniel1** [comment]
After a bunch of changes, I feel ready to call this finished.

**Comment by 4000daniel1** [comment]
!wip

**Review by WickedCybs — CHANGES_REQUESTED** [review]
You should replace the shotgun slugs you mapped in with buckshot. Same reason as the .50 here, It's way too much firepower to a site that doesn't need it, as these are unarmoured simple mobs.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**